guides/doc-Planning_for_Project/topics/Deployment_Scenarios.adoc

```diff
@@ -1,133 +1,3 @@
-[[sect-Architecture_Guide-Multiple_Manifest_Scenarios]]
-=== {ProjectServer} with multiple manifests
-
-If you plan to have more than one Red{nbsp}Hat{nbsp}Network account, or if you want to manage systems belonging to another entity that is also a Red{nbsp}Hat{nbsp}Network account holder, then you and the other account holder can assign subscriptions, as required, to manifests.
-A customer that does not have a {Project} subscription can create a Subscription Asset Manager manifest, which can be used with {Project}, if they have other valid subscriptions.
-You can then use the multiple manifests in one {ProjectServer} to manage multiple organizations.
-
-If you must manage systems but do not have access to the subscriptions for the RPMs, you must use {RHEL} {Project} Add-On.
-For more information, see https://www.redhat.com/en/technologies/management/satellite[{Project} Add-On].
-
-The following diagram shows two Red{nbsp}Hat{nbsp}Network account holders, who want their systems to be managed by the same {Project} installation.
-In this scenario, Example Corporation 1 can allocate any subset of their 60 subscriptions, in this example they have allocated 30, to a manifest.
-This can be imported into the {Project} as a distinct Organization.
-This allows system administrators the ability to manage Example Corporation 1's systems using {Project} completely independently of Example Corporation 2's organizations (R&D, Operations, and Engineering).
-
-[[satellite_server_with_multiple_manifests_image]]
-.{ProjectServer} with multiple manifests
-image::server-multiple-manifests-satellite.png[{ProjectServer} with multiple manifests]
-
-When creating a Red{nbsp}Hat Subscription Manifest:
-
-
-* Add the subscription for {ProjectServer} to the manifest if planning a disconnected or self-registered {ProjectServer}.
-This is not necessary for a connected {ProjectServer} that is subscribed using the Subscription Manager utility on the base system.
-
-* Add subscriptions for all {SmartProxyServers} you want to create.
-
-* Add subscriptions for all Red{nbsp}Hat products you want to manage with {Project}.
-
-* Note the date when the subscriptions are due to expire and plan for their renewal before the expiry date.
-
-* Create one manifest per organization.
-You can use multiple manifests and they can be from different Red Hat subscriptions.
-
-{ProjectName} allows the use of future-dated subscriptions in the manifest.
-This enables uninterrupted access to repositories when future-dated subscriptions are added to a manifest before the expiry date of existing subscriptions.
-
-Note that the Red{nbsp}Hat Subscription Manifest can be modified and reloaded to {ProjectServer} in case of any changes in your infrastructure, or when adding more subscriptions.
-Manifests should not be deleted.
-If you delete the manifest from the Red Hat Customer Portal or in the {ProjectWebUI} it will unregister all of your content hosts.
-endif::[]
-
 [[sect-host-group-structures]]
```

Comment:
Is this intentional? In `guides/doc-Planning_for_Project/master.adoc`, you change this topic to an assembly file. -> it's not in use anymore but still contains some lines?

Reply:
Good catch, thank you. This isn't supposed to be here anymore. I deleted the whole file.